3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,109 @@
+//Flight booking fullname function && Formal fullname
+function getFullname(firstname, surname, sex, useFormalName = false) {
+  if (useFormalName === true) {
+    if (sex === "woman") {
+      return `lady ${firstname} ${surname}`;
+    } else if (sex === "man") {
+      return `Lord ${firstname} ${surname}`;
+    }
+  } else {
+    return `${firstname} ${surname}`;
+  }
+}
+const fullname1 = getFullname("Benjamin", "Hughes", "woman", true);
+const fullname2 = getFullname("hadis", "R", "man", false);
+const fullname3 = getFullname("hadis", "R");
+
+//Event application
```

Comment:
Your code is generally fine and right, but you can improve here by using new Date() function and also an array, where you declare all days, like 'Monday', 'Tuesday' etc.

`function getEventWeekday(daysToAdd) {
  const today = new Date();
  const futureDate = new Date(today);
  futureDate.setDate(today.getDate() + daysToAdd);

  const daysOfWeek = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
  const eventDayOfWeek = daysOfWeek[futureDate.getDay()];

  return eventDayOfWeek;
}

const eventApplication = getEventWeekday(3);
console.log(eventApplication);`

In this version, `getEventWeekday` takes a number of days to add, creates a Date object representing today's date, adds the specified number of days, and then retrieves the day of the week using getDay(). The result is a string representing the day of the week.
This approach makes the code more flexible and easier to understand, and it avoids hardcoding day numbers :)